modules/temp-channels/channel-settings.js

```diff
@@ -0,0 +1,282 @@
+                interaction.editReply(localize('temp-channels', 'edit-error'));
+                return;
+            }
+            vcLimit = interaction.options.getInteger('user-limit');
+            edited++;
+        } else vcLimit = vchann.userLimit;
+        if (interaction.options.getInteger('bitrate')) {
+            if (interaction.options.getInteger('bitrate') <= 8000 || interaction.options.getInteger('bitrate') >= interaction.guild.maximumBitrate) {
+                interaction.editReply(localize('temp-channels', 'edit-error'));
+                return;
+            }
+            vcBitrate = interaction.options.getInteger('bitrate');
+            edited++;
+        } else vcBitrate = vchann.bitrate;
+        if (interaction.options.getString('name')) {
+            vcName = interaction.options.getString('name');
+            edited++;
+        } else vcName = vchann.name;
+        if (interaction.options.getBoolean('nsfw')) {
+            vcNsfw = interaction.options.getBoolean('nsfw');
+            edited++;
+        } else vcNsfw = vchann.nsfw;
+    }
+    if (callerInfo === 'modal') {
+        if (interaction.fields.getTextInputValue('edit-modal-limit-input') >= 0) {
+            if (interaction.fields.getTextInputValue('edit-modal-limit-input') < 0 || interaction.fields.getTextInputValue('edit-modal-limit-input') > 99) {
+                interaction.editReply(localize('temp-channels', 'edit-error'));
+                return;
+            }
+            vcLimit = interaction.fields.getTextInputValue('edit-modal-limit-input');
+            edited++;
+        } else vcLimit = vchann.userLimit;
+        if (interaction.fields.getTextInputValue('edit-modal-bitrate-input')) {
+            if (interaction.fields.getTextInputValue('edit-modal-bitrate-input') <= 8000 || interaction.fields.getTextInputValue('edit-modal-bitrate-input') >= interaction.guild.maximumBitrate) {
+                interaction.editReply(localize('temp-channels', 'edit-error'));
+                return;
+            }
+            vcBitrate = interaction.fields.getTextInputValue('edit-modal-bitrate-input');
+            edited++;
+        } else vcBitrate = vchann.bitrate;
+        if (interaction.fields.getTextInputValue('edit-modal-name-input')) {
+            vcName = interaction.fields.getTextInputValue('edit-modal-name-input');
+            edited++;
+        } else vcName = vchann.name;
+        if (interaction.fields.getTextInputValue('edit-modal-nsfw-input')) {
+            const nsfwInput = interaction.fields.getTextInputValue('edit-modal-nsfw-input');
+            vcNsfw = (nsfwInput === 'true');
+            edited++;
+        } else vcNsfw = vchann.nsfw;
+    }
+
+    if (edited !== 0) {
+        interaction.editReply(await embedType(moduleConfig['channelEdited'], {}, {ephemeral: true}));
+        try {
+            vchann.edit({userLimit: vcLimit, nsfw: vcNsfw, name: vcName, bitrate: vcBitrate});
```

Comment:
Editing the name via the modal does not work for me at all

Reply:
I have no idea why, but it works for me

Reply:
yeah idk works for me too now :sweat_smile: